corehq/apps/locations/static/locations/ko/location.js

```diff
@@ -0,0 +1,33 @@
+// for product and user per location selection
+$(function () {
+    $("#id_users-selected_ids").width(800).height(400).multiselect();
+    $("#id_products-selected_ids").width(800).height(400).multiselect();
```

Comment:
It's a relic of the [multiselect widget](https://github.com/michael/multiselect) we're using - it requires that the element have a fixed size in order to render properly.  These two lines are moved from the original template, now that I've added a standalone js file for this page.